apps/omg_watcher/lib/omg_watcher/db/txoutput.ex

```diff
@@ -112,13 +116,16 @@ defmodule OMG.Watcher.DB.TxOutput do
     }
   end
 
+  @decorate measure_event()
```

Comment:
isn't this already handled? https://hexdocs.pm/appsignal/Appsignal.Ecto.html

Reply:
a bit yes,
but it seems to me that in ecto there is no time for processing into structs (creating query, and transform to result), only the time of working on the database